Neos.Neos/Classes/Domain/Repository/SiteRepository.php

```diff
@@ -56,6 +56,36 @@ class SiteRepository extends Repository
      */
     protected $defaultSiteNodeName;
 
+    public function findOnline(): iterable
+    {
+        return $this->findAll();
+    }
+
+    public function findAll(): iterable
+    {
+        $cr = $this->contentRepositoryRegistry->get(ContentRepositoryId::fromString('default'));
```

Comment:
how can we find all site node aggregates efficiently? Loop over all crs registered and fetch the node aggregates und the `Neos.Neos:Sites` node?